css/accordion.css

```diff
@@ -71,6 +71,13 @@ html {
   padding: 0.5rem 1.2rem;
 }
 
+.acc input:checked ~ label .rotate {
```

Comment:
@JoeStanton I can't! Please help 🙏 

I did manage to replace the jquery though 👏